packages/relayer/message/process_message.go

```diff
@@ -165,3 +172,48 @@ func (p *Processor) sendProcessMessageCall(
 func (p *Processor) setLatestNonce(nonce uint64) {
 	p.destNonce = nonce
 }
+
+func (p *Processor) saveMessageStatusChangedEvent(
+	ctx context.Context,
+	receipt *types.Receipt,
+	e *relayer.Event,
+	event *bridge.BridgeMessageSent,
+) error {
+	bridgeAbi, err := abi.JSON(strings.NewReader(bridge.BridgeABI))
+	if err != nil {
+		return errors.Wrap(err, "abi.JSON")
+	}
+
+	m := make(map[string]interface{})
+
+	for _, log := range receipt.Logs {
+		topic := log.Topics[0]
+		if topic == crypto.Keccak256Hash([]byte("MessageStatusChanged(bytes32,uint8,address)")) {
```

Comment:
Can we re-use the values defined in `bridgeAbi.Events[idx].Name / bridgeAbi.Events[idx].ID` here? instead of maintaining the hard-coded event selectors manually here?

Reply:
Yep, will do!